docs/design-notes/per-partition-rate-limit.md

```diff
@@ -0,0 +1,229 @@
+# Per-partition rate limiting
+
+Scylla clusters operate best when the data is spread across a large number
```

Comment:
Please also add user-facing documentation in cql-extensions.md.

Reply:
Done